.github/workflows/release_pypi.yml

```diff
@@ -0,0 +1,104 @@
+name: PyPI Release
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  macos:
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.6
+      - name: Install Rust toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          target: aarch64-apple-darwin
+          profile: minimal
+          default: true
+      - name: Install maturin
+        run: pip install maturin
```

Comment:
I prefer to have all dependencies pinned using the `pip-compile --generate-hashes` tool from `pip-tools` and then use `pip install -r requirements.txt`.

Feel free to do this in this PR. Or I'll do it as a follow-up.

Reply:
Failed to get it work for both macOS, Windows and Linux so I've reverted back to just `pip install maturin`.